### DISCUSSION TITLE
add humidity to climate card and history

### DISCUSSION THREAD

**Discussion by oliverrahner** [home-assistant/frontend#14260] [Dashboard]
I have some thermostats for floor heating, which also provide data on current humidity. The integration puts that info into the climate entity correctly:
![image](https://user-images.githubusercontent.com/2277681/199536695-2b09ef54-f5ce-4563-b429-5c23e3ddaa39.png)

But I see that data neither on the dashboard card, nor in the device's info and, consequently, also not in the History, where I would like to have it most of all.

Dashboard Card:
![image](https://user-images.githubusercontent.com/2277681/199537789-49616f08-4ce4-4ec0-a82d-09da1f793b3c.png)

Info:
![image](https://user-images.githubusercontent.com/2277681/199537848-e2991879-cf7f-48b1-be52-71453afdeaf3.png)

History:
![image](https://user-images.githubusercontent.com/2277681/199537937-45b26319-eb80-4679-a3f0-f3fd59a5468b.png)

History:

**Comment by adejong5** [comment]
I was also looking for this feature. I found issue #4740, which seems to have been erroneously closed. Humidity is part of the climate entity and should be visible in the thermostat card.

**Reply by adejong5** [reply]
Any comment from @zsarnett on why he closed #4740? The closing comment seems to confuse which card/issue was being addressed.
